package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "onboard-taqtile",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
```

Geralmente nos projetos usamos versões especificas da lib, para evitar problemas com braking changes e imcompatibilidades futuras.
assim:
```suggestion
    "react": "18.3.1",
```

poderia aplicar para todas as libs e reinstalar para que atualize o package-look

---

Resolvido!